paddle/fluid/inference/api/paddle_pass_builder.cc

```diff
@@ -348,6 +348,9 @@ void CpuPassStrategy::EnableMkldnnQuantizer() {
 void CpuPassStrategy::EnableMkldnnBfloat16() {
 #ifdef PADDLE_WITH_MKLDNN
   if (!use_mkldnn_bfloat16_) {
+    passes_.push_back("fc_mkldnn_pass");
+    passes_.push_back("fc_act_mkldnn_fuse_pass");
+
```

Comment:
```suggestion
passes_.push_back("fc_elementwise_add_mkldnn_fuse_pass");

```
I think we can get even better results by adding `fc_elementwise_add_mkldnn_fuse_pass` here. For now it's not working, because `fc_mkldnn_pass` is disabled by default in _paddle_pass_builder.cc_. Could you add this line and run benchmark one more time?

Reply:
Test in progress...

Reply:
I tried to enable it but unfortunately it caused a segmentation fault in bfloat16 pass.
Disabled for now.

Reply:
I confirmed that this change speed-up VIT-OCR on bfloat16 significantly.